src/components/typography/typography.tsx

```diff
@@ -0,0 +1,36 @@
+import { useTheme } from '@react-navigation/native';
+import React from 'react';
+import { Text, TextProps } from 'react-native';
+
+import { styles } from './typography.styles';
+
+interface TypographyProps extends TextProps {
+  variant?: 'h1' | 'h2' | 'h3' | 'body' | 'caption';
```

Comment:
Can you add in comment a reference point where this is coming from aka which library we are using as a base reference? BaseWeb / Tailwaind etc?